tensorflow/python/keras/layers/convolutional.py

```diff
@@ -303,6 +314,19 @@ def compute_output_shape(self, input_shape):
   def _recreate_conv_op(self, inputs):  # pylint: disable=unused-argument
     return False
 
+  def _check_invalid_dimension(self, dimension, idx):
+    """Checks if output has all positive dimensions"""
+    output_dimension = conv_utils.conv_output_length(dimension,
+                                                     self.kernel_size[idx],
+                                                     self.padding,
+                                                     self.strides[idx],
+                                                     dilation = self.dilation_rate[idx])
+    if output_dimension is not None and output_dimension<=0:
```

Comment:
Nit: space between var and operator: output_dimension<=0 => output_dimension <= 0